### PR TITLE
[INGEST] added Ritzau config panel

### PR DESCRIPTION
**Pull request by jerome-poisson**
SDESK-2328

### CODE REVIEW

File: scripts/apps/ingest/index.js [hunk 1]
```diff
@@ -201,4 +201,12 @@ angular.module('superdesk.apps.ingest', [
             label: 'Wufoo Forms',
             templateUrl: asset.templateUrl('apps/ingest/views/settings/wufoo.html')
         });
+        ingestSources.registerFeedingService('ritzau', {
+            label: 'Ritzau Webservice',
+            templateUrl: asset.templateUrl('apps/ingest/views/settings/ritzauConfig.html')
+        });
+        ingestSources.registerFeedingService('ntb_ritzau', {
```
Comment (petrjasek) [inline scripts/apps/ingest/index.js]: is there anything ntb specific on the service?

Comment (jerome-poisson) [reply scripts/apps/ingest/index.js]: just small stuff, the same that we do for other feeds (default category, utenriks or sport), but still it is a specific behaviour.

Comment (petrjasek) [reply scripts/apps/ingest/index.js]: isn't it done with the parser?

Comment (jerome-poisson) [reply scripts/apps/ingest/index.js]: Oh right, got confused between service and parser :-/ . Will fix that thanks!

**Pull request merged**